elm-format-lib/package.yaml

```diff
@@ -23,24 +23,18 @@ default-extensions:
 ghc-options:
   - -O2
   - -Wall
+  - -Werror=unused-packages
   - -Wno-name-shadowing
 
 dependencies:
   aeson: ">= 2.1.0.0 && < 2.2"
-  ansi-terminal: ">= 0.11 && < 0.12"
-  ansi-wl-pprint: ">= 0.6.9 && < 0.7"
-  array: ">= 0.5.4.0 && < 0.6"
   base: ">= 4.15.0.0 && < 5"
   bimap: ">= 0.4.0 && < 0.6"
   binary: ">= 0.8.9.0 && < 0.9"
   bytestring: ">= 0.11.3.0 && < 0.12"
   containers: ">= 0.6.5.1 && < 0.7"
-  directory: ">= 1.3.6.2 && < 2"
-  filepath: ">= 1.4.2.1 && < 2"
-  ghc-prim: ">= 0.5.2"
```

Comment:
@8n8 Do you remember why you had to re-add ghc-prim in your initial PR?  It made me remove it again when I made the warning into an error, and it compiled fine for me locally.  I guess I'll see if CI fails.

Reply:
Yes, I removed it when I saw the warning, but then when I built again from scratch it failed. I think I had to do `git clean -f` to remove the build caches to make it fail.

Reply:
If you replace `import GHC.Prim` with `import GHC.Exts` in `elm-format-lib/src/Data/Utf8.hs` it fixes the issue.

The [docs](https://hackage.haskell.org/package/ghc-prim) for GHC.Prim say:

> This package contains the primitive types and operations supplied by GHC. It is an internal package, only for the use of GHC developers. GHC users should not use it! If you do use it then expect breaking changes at any time without warning. You should prefer to import GHC.Exts from the base package instead.

Reply:
Ah, great!  I'll make the change to `GHC.Exts`.  I also reported [#23212: -Werror=unused-packages cannot be satisfied for ghc-prim · Issues · Glasgow Haskell Compiler / GHC · GitLab](https://gitlab.haskell.org/ghc/ghc/-/issues/23212).

Reply:
oh, also `dev/build.sh clean && dev/build.sh` was enough to trigger the original problem without having to re-clone.

Reply:
Oh right. That's easier, thanks.